how-to-guides/KyloTemplatesDocs.rst

```diff
@@ -0,0 +1,114 @@
+======================
+Kylo Templates Guide
```

Comment:
Should include this part http://kylo.readthedocs.io/en/v0.8.2/user-guides/KyloOperationsGuide.html#migrating-templates-and-feeds
And maybe move it under User guides later on, when we split stuff